src/Core/src/PublicAPI/net-ios/PublicAPI.Shipped.txt

```diff
@@ -2255,6 +2255,8 @@ override Microsoft.Maui.Platform.MauiTextView.LayoutSubviews() -> void
 override Microsoft.Maui.Platform.MauiTextView.MovedToWindow() -> void
 override Microsoft.Maui.Platform.MauiTextView.Text.get -> string?
 override Microsoft.Maui.Platform.MauiTextView.Text.set -> void
+override Microsoft.Maui.Platform.MauiTextView.TextAlignment.get -> UIKit.UITextAlignment
```

Comment:
These should be in unshipped.txt

Reply:
@PureWeen , I have moved the code changes to unshipped.txt.